docs/validators/cosmovisor.mdx

```diff
@@ -0,0 +1,154 @@
+hide_title: false
+id: setting-up-Cosmovisor
+title: Setting up Cosmovisor
+---
+
+# Setting up Cosmovisor
+
+## Installing Cosmovisor
+
+```
+go install cosmossdk.io/tools/cosmovisor/cmd/cosmovisor@v1.5.0
+mv $GOPATH/bin/cosmovisor /usr/local/bin/cosmovisor
+```
+
+### Download Cosmovisor Binary
+
+Download the Binary for your system architecture from the following links and
+place it in your $PATH
+amd64: https://github.com/cosmos/cosmos-sdk/releases/download/cosmovisor%2Fv1.5.0/cosmovisor-v1.5.0-linux-amd64.tar.gz
+arm64: https://github.com/cosmos/cosmos-sdk/releases/download/cosmovisor%2Fv1.5.0/cosmovisor-v1.5.0-linux-arm64.tar.gz
+
+This example assumes you are using amd64 architecture. If you are using arm64
+replace amd64 with arm64 in the commands below.
+
+```bash
+wget https://github.com/cosmos/cosmos-sdk/releases/download/cosmovisor%2Fv1.5.0/cosmovisor-v1.5.0-linux-amd64.tar.gz
+mv cosmovisor-amd64 /usr/local/bin/cosmovisor
+```
+
+## Setting up Cosmovisor
+
+### Create `Cosmovisor` Directories
+
+```bash
+mkdir -p ~/.zetacored/config
+mkdir -p ~/.zetacored/cosmovisor/genesis/bin
+mkdir -p ~/.zetacored/cosmovisor/upgrades
+
+```
+
+### Setup `Cosmovisor` systemd service
+
+In this example we'll run Cosmovisor as a systemd service. You can also run it as
+a docker container or as a standalone process. The environment variables are
+explained in the [Environment Variables](#environment-variables) section at the
+end of this document.
+
+Create a file at `/etc/systemd/system/cosmovisor.service` with the following
+contents. Make sure to change the `User` and `/home/zetachain/` fields to the
+user you want to run the service as.
+
+```bash
+[Unit]
+Description=Cosmovisor ZetaChain Service
+After=multi-user.target
+StartLimitIntervalSec=0
+[Install]
+WantedBy=multi-user.target
+[Service]
+WorkingDirectory=/home/zetachain/.zetacored/cosmovisor
+Environment="DAEMON_HOME=/home/zetachain/.zetacored"
+Environment="DAEMON_NAME=zetacored"
+Environment="DAEMON_ALLOW_DOWNLOAD_BINARIES=false"
+Environment="DAEMON_RESTART_AFTER_UPGRADE=true"
+Environment="DAEMON_DATA_BACKUP_DIR=/home/zetachain/.zetacored"
+Environment="CLIENT_DAEMON_NAME=zetaclientd"
+Environment="CLIENT_SKIP_UPGRADE=false"
+Environment="CLIENT_START_PROCESS=false"
+Environment="UNSAFE_SKIP_BACKUP=true"
+Type=simple
+Restart=always
+RestartSec=600
+User=zetachain
+LimitNOFILE=262144
+ExecStart=cosmovisor start --home /home/zetachain/.zetacored/ --log_format json  --moniker <YOUR_NODE_NAME_HERE>
+```
+
+## Install the `zetacored` Binary
+
+### Download `zetacored` Binary
+
+Download the latest `zetacored` binary for your system architecture from the following
+links and place it in the `cosmovisor/genesis/bin/` directory. This example
+assumes you are using ubuntu 22 on amd64 architecture. You may need to make
+adjustments for your system.
+
+```bash
+wget https://zetachain-external-files.s3.amazonaws.com/binaries/athens3/latest/zetacored-ubuntu-22-amd64
```

Comment:
Replace this link with https://github.com/zeta-chain/node/releases/download/latest/zetacored-ubuntu-22-amd64